README.md

```diff
@@ -157,60 +157,99 @@ speakeasy usage [flags]
-  * [ ] `deepObject` style serialization
-  * [ ] Other styles not currently supported
-* [ ] Request Headers - <https://swagger.io/docs/specification/serialization/#header>
-  * [ ] Including explode handling
-* [ ] Request Body Serialization
-  * [ ] Multipart Encoding - <https://swagger.io/docs/specification/describing-request-body/multipart-requests/>
-    * [ ] Binary file support
-    * [ ] Form data support
-    * [ ] Encoding not supported
-  * [ ] JSON Serialization
-  * [ ] x-www-form-urlencoded Serialization - <https://swagger.io/docs/specification/describing-request-body>
-    * [ ] Including encoding
-    * [ ] Doesn't support non-object types
-  * [ ] plain text / string serialization
-  * [ ] raw byte serialization
-  * [ ] Other serialization not currently supported
-  * [ ] Handling of `required` body
-* [ ] Response Body Serialization
-  * [ ] Return StatusCode and Content-Type
-  * [ ] plain text / string deserialization
-  * [ ] raw byte deserialization
-  * [ ] Json deserialization
-  * [ ] Other deserialization not currently supported
-* [ ] Media-type patterns - <https://swagger.io/docs/specification/media-types/>
-* [ ] Full OpenAPI datatype support
-  * [ ] Basic types - <https://swagger.io/docs/specification/data-models/data-types/>
-  * [ ] Enums
-  * [ ] Number formats ie float, double, int32, int64
-  * [ ] Date-time
-  * [ ] Binary
-  * [ ] Arrays
-  * [ ] Objects
-  * [ ] Optional
-  * [ ] Maps
-  * [ ] Any type
-  * [ ] OneOf/AnyOf/AllOf - <https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/>
-* [ ] Auxiliary files
-  * [ ] Utilities classes/functions to help with serialization/deserialization.
-  * [ ] Files needed for creating a fully compilable package that can be published to the relevant package manager without further changes.
-* [ ] Support for x-speakeasy-server-id map generation
-* [ ] Support for snippet generation
-* [ ] Support for readme generation
+The tables below give an overview of what we support from OpenAPI.
+
+### ✅ Server Configuration 
+
+| Name | Notes | Swagger Link |Support |
+|------|-------|--------------|--------|
+| ServerURL config | Global and per method (include base url and templating) | [api-host-and-base-path](https://swagger.io/docs/specification/api-host-and-base-path/) |✅ |
+| Authentication and Security | Global and per method | [authentication](https://swagger.io/docs/specification/authentication/) |✅ |
+| Method Generation |  |   |✅ |
+| Model Generation |  Request and Response |   |✅ |
+
+### ⚠️ Path Paramamters Serialization ([path-parameters](https://swagger.io/docs/specification/describing-parameters/#path-parameters))
+
+| Name | Notes | Swagger Link | Support |
+|------|-------|--------------|---------|
+| Default | `(style = simple, explode = false)` | [serialiszation/#path](https://swagger.io/docs/specification/serialization/#path/) | ✅ |
+| Basic types | | | ✅ |
+| Simple objects | | | ✅ |
+| Other styles | | | ❌ |
```

Comment:
Can we replace `Other styles` with two rows, one for `label` and one for `matrix` (both not supported)